sdk/modelsrepository/azure-iot-modelsrepository/build.ps1

```diff
@@ -0,0 +1,15 @@
+# Builds the azure-iot-modelsrepository project
+# The output will be in the target directory.
+# You can use the generated azure-iot-modelsrepository-x-sources.jar to upload to the API review tool at https://apiview.dev/
+
+param (
+    [Parameter(Mandatory = $false)]
+    [switch] $skipTests
+)
+
+if ($skipTests) {
+    mvn install -DskipTests
+}
```

Comment:
shouldn't this be on the same line as `else {` ?

Reply:
sure, updated.